schemas/top-api.yml

```diff
@@ -193,6 +195,24 @@ paths:
           $ref: '#/components/responses/UnexpectedError'
       tags:
         - document
+  '/document/query/{organisationId}/{repositoryId}/{queryId}':
```

Comment:
Is this path still used? I don't see the difference to `getQueryResultIds`.

Reply:
If you make any changes to address this comment please note that I already rebased your branch to have a consistent history.

Reply:
it's a little messy, but they return two different results (one DocumentPages, the other a list of ids). I renamed the former to be less confusing. Maybe this needs to be consolidated in the future?

Reply:
> If you make any changes to address this comment please note that I already rebased your branch to have a consistent history.

Something is not right with my rebase workflow. I rebased my branch as well before the commit (at least I thought so). And it always seems like a real pain.